packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -82,6 +99,11 @@ export type VisualizeSharedObject = (
  */
 export type VisualizeChildData = (data: unknown) => Promise<VisualChildNode>;
 
+/**
+ * A visual representation of {@link @fluidframework/shared-object-base#ISharedObject} & {@link DataObject} data.
```

Comment:
This type is just the shared object or data object themselves, it's not a visual representation of either yet.
```suggestion
 * Utility type for a union of things that can be visualized.
```